lib/elastomer/client/docs.rb

```diff
@@ -261,6 +261,38 @@ def delete_by_query(query, params = nil)
         @client.delete_by_query(query, update_params(params))
       end
 
+      # Matches a provided or existing document to the stored percolator
+      # queries. To match an existing document, pass `nil` as the body and
+      # include `:id` in the params.
+      #
+      # Examples
+      #
+      #   index.register_percolator_query 1, :query => { :match => { :author => "pea53" } }
+      #   docs.percolate :doc => { :author => "pea53" }
+      #   docs.percolate nil, :id => 3
+      #
+      # Returns the response body as a Hash
+      def percolate(body = nil, params = {})
```

Comment:
Since a percolate request with no `body` or `params` is invalid, I think `body` should not have a default value. Same for `percolate_count`.